src/genpy/message.py

```diff
@@ -56,15 +57,32 @@
 except NameError:  # Python 3
     from importlib import reload
 
-# common struct pattern singletons for msgs to use. Although this
-# would better placed in a generator-specific module, we don't want to
-# add another import to messages (which incurs higher import cost)
-
 if sys.version > '3':
     long = int
 
+# common struct pattern singletons for msgs to use. Although this
+# would better placed in a generator-specific module, we don't want to
+# add another import to messages (which incurs higher import cost)
```

Comment:
Please avoid moving this comment around if there is no reason.

Reply:
I thought it was obvious that it was meant to be with the `struct_I = struct.Struct('<I')` line, and that the py3 hack was accidentally inserted between them?

Reply:
Ok, since this repo uses a single branch for all active distros (so no need to port patches to other branches) it can stay in this PR.

Reply:
.... perhaps not, based on the blame for the file. I shall revert this change.